packages/invoices/src/OutboundShipment/DetailView/index.ts

```diff
@@ -24,11 +24,13 @@ export const placeholderInvoice: OutboundShipment = {
   pickedDatetime: '',
   deliveredDatetime: '',
 
-  name: {
-    id: '',
-    name: '',
-    code: '',
-    isCustomer: true,
-    isSupplier: false,
-  },
+  purchaseOrderNumber: undefined,
+  goodsReceiptNumber: undefined,
+  requisitionNumber: undefined,
+  inboundShipmentNumber: undefined,
+
+  transportReference: undefined,
+  shippingMethod: undefined,
+
+  otherParty: undefined,
```

Comment:
Not ideal - the draft when constructed is picking values from the grpahql response by what is in this object